types/popper.js/index.d.ts

```diff
@@ -1,58 +1,75 @@
-// Type definitions for popper.js v0.4.0
+// Copyright 2017 Canva Inc. All Rights Reserved.
```

Comment:
This copyright notice should be removed because all files are served with license at `DefinitelyTyped/LICENSE`.